ern-local-cli/src/lib/utils.js

```diff
@@ -504,32 +504,70 @@ async function launchAndroidRunner (pathToAndroidRunner: string) {
 }
 
 async function launchIosRunner (pathToIosRunner: string) {
-  const iPhoneDevice = await ios.askUserToSelectAniPhoneDevice()
+  const iosDevices = ios.getiPhoneRealDevices()
+  if (iosDevices && iosDevices.length > 0) {
+    log.debug(`Will launch ErnRunner on a real device`)
```

Comment:
minor: use single quote instead of backticks for pure string (non template litteral)

Reply:
please see if you like ternary representation? No logical change. 

iosDevices && iosDevices.length > 0 ? launchOnDevice(pathToIosRunner, iosDevices) : launchOnSimulator(pathToIosRunner)

moving the log statement to the underlying class

Reply:
@krunalsshah flow doesn't like this syntax. Cannot pass flow check with ternary.

Reply:
@clairewli the syntax below is usually okay with standard, although skip the ternary part as it doesn't lead to any optimization other than syntactic representation

```
iosDevices && iosDevices.length > 0 
      ? launchOnDevice(pathToIosRunner, iosDevices) 
      : launchOnSimulator(pathToIosRunner)
```